.github/workflows/integrate.yaml

```diff
@@ -47,16 +47,14 @@ jobs:
         with:
           provider: microk8s
           channel: 1.24/stable
-          # Pinned until this bug is resolved: https://bugs.launchpad.net/juju/+bug/1992833
-          bootstrap-options: "--agent-version=2.9.34"
+          juju-channel: 2.9/stable
```

Comment:
This is now the standard in our repositories, but I think this change is not related to bumping SDI-> 0.3.6. Also, I think we should not change it to preserve the state of the CI as it was back when we cut the branch, unless it is giving us trouble.

Reply:
Will revert.